components/list/list.css

```diff
@@ -0,0 +1,70 @@
+/* @import url('../../components/root.css'); */
+
+@import url('../../components/framer.css');
+
+
+.list-bg{
+    background-color: var(--text-col-white);
+    border-radius: 0.313rem;
+}
+
+.text-deocration-none{
+    text-decoration: none;
+}
+.list-style-none{
+    list-style: none;
+}
+
+.list-inline{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+}
+.list-square{
+    list-style-type: square;
+    margin-bottom: 0.75rem;
+}
+
+.list-circle{
+    list-style-type: circle;
+    margin-bottom: 0.75rem;
+
+}
+
+.list-disc{
+    list-style-type: disc;
+    margin-bottom: 0.75rem;
+
```

Comment:
Use px here

Reply:
well 0.75 if you convert it into px it is actually 12px